apis/config/v1beta1/defaults.go

```diff
@@ -29,16 +29,17 @@ import (
 )
 
 const (
-	DefaultNamespace                      = "kueue-system"
-	DefaultWebhookServiceName             = "kueue-webhook-service"
-	DefaultWebhookSecretName              = "kueue-webhook-server-cert"
-	DefaultWebhookPort                    = 9443
-	DefaultHealthProbeBindAddress         = ":8081"
-	DefaultMetricsBindAddress             = ":8080"
-	DefaultLeaderElectionID               = "c1f6bfd2.kueue.x-k8s.io"
-	DefaultClientConnectionQPS    float32 = 20.0
-	DefaultClientConnectionBurst  int32   = 30
-	defaultPodsReadyTimeout               = 5 * time.Minute
+	DefaultNamespace                                    = "kueue-system"
+	DefaultWebhookServiceName                           = "kueue-webhook-service"
+	DefaultWebhookSecretName                            = "kueue-webhook-server-cert"
+	DefaultWebhookPort                                  = 9443
+	DefaultHealthProbeBindAddress                       = ":8081"
+	DefaultMetricsBindAddress                           = ":8080"
+	DefaultLeaderElectionID                             = "c1f6bfd2.kueue.x-k8s.io"
+	DefaultClientConnectionQPS                  float32 = 20.0
+	DefaultClientConnectionBurst                int32   = 30
+	defaultPodsReadyTimeout                             = 5 * time.Minute
+	DefaultQueueVisibilityUpdateIntervalSeconds         = 5
```

Comment:
```suggestion
	DefaultQueueVisibilityUpdateIntervalSeconds  int64 = 5
```

Reply:
using int32 won't be enough? In your API we are using int32

Reply:
Oops, yes